expression/integration_test/integration_test.go

```diff
@@ -2059,3 +2059,15 @@ func TestIssue41986(t *testing.T) {
 	// shouldn't report they can't find column error and return the right result.
 	tk.MustQuery("SELECT GROUP_CONCAT(effective_date order by stlmnt_hour DESC) FROM ( SELECT (COALESCE(pct.clearing_time, 0)/3600000) AS stlmnt_hour ,COALESCE(pct.effective_date, '1970-01-01 08:00:00') AS effective_date FROM poi_clearing_time_topic pct ORDER BY pct.effective_date DESC ) a;").Check(testkit.Rows("2023-08-25 00:00:00"))
 }
+
+func TestCastErrMsg(t *testing.T) {
```

Comment:
Better move it to `tests/integrationtest`. It will have better performance.

Reply:
done